src/Transactions/TransactionFields/LinesField.php

```diff
@@ -32,23 +48,45 @@ public function getLines(): array
          * one or more detail lines and optionally one or more vat lines. Twinfield returns an error when the lines are
          * in an incorrect order.
          */
-        $type_order = [LineType::TOTAL(), LineType::DETAIL(), LineType::VAT()];
-        usort(
-            $this->lines,
-            function (TransactionLine $a, TransactionLine $b) use ($type_order): int
-            {
-                $type_index_a = array_search($a->getLineType(), $type_order, false);
-                $type_index_b = array_search($b->getLineType(), $type_order, false);
-
-                if ($type_index_a != $type_index_b) {
-                    return $type_index_a <=> $type_index_b;
-                }
-
-                return $a->getId() <=> $b->getId();
-            }
-        );
-
-        return $this->lines;
+        $lines = [];
```

Comment:
```php
return array_merge(
    $this->getTotalLines(),
    $this->getDetailLines(),
    $this->getVatLines()
);
```

Reply:
Ahah. Just re-read the docs of `array_merge()` and that would indeed work! I figured it wouldn't work because it overrides values with the same key, but it only does that for string keys, not for integer keys!